app/src/main/java/com/github/dedis/student20_pop/MainActivity.java

```diff
@@ -66,17 +66,19 @@ public void onClick(View view) {
                 break;
             case R.id.button_launch:
                 String name = ((EditText) findViewById(R.id.entry_box_launch)).getText().toString();
-                // For later: send LAO and organizer information
                 Person organizer = new Person("name");
```

Comment:
`organizer =  ((PoPApplication) getApplication()).getPerson()`
We should extract organizer as us here.

Reply:
Indeed, good catch